source/MRViewer/ImGuiHelpers.cpp

```diff
@@ -685,4 +685,22 @@ PaletteChanges Palette(
     return PaletteChanges( changes );
 }
 
+void Image( const MR::ImGuiImage& image, const ImVec2& size, const MR::Color& multColor )
+{
+    MR::Vector4 tintColor = ( MR::Vector4f ) multColor;
+    Image( image, size, ImVec4( tintColor.x, tintColor.y, tintColor.z, tintColor.w ) );
+}
+
+void Image( const MR::ImGuiImage& image, const ImVec2& size, const ImVec4& multColor )
+{
+    Image( image.getImTextureId(), size, ImVec2( 0, 1 ), ImVec2( 1, 0 ), multColor );
+}
+
+MR::Vector2i GetImagePointerCoord( const MR::ImGuiImage& image, const ImVec2& size)
+{
+    auto pos = ImGui::GetCursorScreenPos();
+    const auto& io = ImGui::GetIO();
+    return  { int( ( io.MousePos.x - pos.x ) / size.x * image.getImageWidth() ), int( ( -io.MousePos.y + pos.y - ImGui::GetStyle().WindowPadding.y ) / size.y * image.getImageHeight() ) };
```

Comment:
why use WindowPadding in Y and do not use it in X, and what if this is called not right after Image,
seems like Image position required as argument